examples/step-67/step-67.cc

```diff
@@ -1465,6 +1501,288 @@ namespace Euler_DG
+                                evaluate_function(*subsonic_outflow_boundaries
+                                                     .find(boundary_id)
+                                                     ->second,
+                                                  phi_m.quadrature_point(q),
+                                                  dim + 1);
+                              at_outflow = true;
+                            }
+                          else
+                            AssertThrow(false,
+                                        ExcMessage(
+                                          "Unknown boundary id, did "
+                                          "you set a boundary condition for "
+                                          "this part of the domain boundary?"));
+
+                          auto flux =
+                            euler_numerical_flux<dim>(w_m, w_p, normal);
+
+                          if (at_outflow)
+                            for (unsigned int v = 0;
+                                 v < VectorizedArray<Number>::size();
+                                 ++v)
+                              {
+                                if (rho_u_dot_n[v] < -1e-12)
+                                  for (unsigned int d = 0; d < dim; ++d)
+                                    flux[d + 1][v] = 0.;
+                              }
+
+                          phi_m.submit_value(-flux, q);
+                        }
+                    }
+
+                  internal::FEFaceNormalQuadratureEvaluationImpl<
+                    dim,
+                    n_points_1d,
+                    dim + 2,
+                    VectorizedArray<Number>>::template interpolate<false,
+                                                                   true>(
+                    data.get_shape_info(),
+                    phi_m.begin_values(),
+                    phi.begin_values(),
+                    false,
+                    face);
+                }
+
+              for (unsigned int q = 0; q < phi.static_n_q_points; ++q)
+                {
+                  const auto factor = VectorizedArray<Number>(1.0) / phi.JxW(q);
+                  for (unsigned int c = 0; c < dim + 2; ++c)
+                    phi.begin_values()[c * phi.static_n_q_points + q] =
+                      phi.begin_values()[c * phi.static_n_q_points + q] *
+                      factor;
+                }
+
+              internal::FEEvaluationImplBasisChange<
+                dealii::internal::EvaluatorVariant::evaluate_evenodd,
+                dim,
+                degree + 1,
+                n_points_1d,
+                VectorizedArray<Number>,
+                VectorizedArray<Number>>::
+                do_backward_hessians(
+                  dim + 2,
+                  data.get_shape_info().data[0].inverse_shape_values_eo,
+                  false,
+                  phi.begin_values(),
+                  phi.begin_dof_values());
+
```

Comment:
Isn't this https://github.com/dealii/dealii/blob/e9461deb2e453f353d6bcab7d571aa2d4715ca47/include/deal.II/matrix_free/evaluation_kernels.h#L3238-L3272